modules/rte_mis_core/src/Form/OverrideTermForm.php

```diff
@@ -81,4 +81,39 @@ public function customSubmit(array $form, FormStateInterface $form_state) {
 
   }
 
+  /**
+   * {@inheritdoc}
+   */
+  public function save(array $form, FormStateInterface $form_state) {
+    $form = parent::buildForm($form, $form_state);
```

Comment:
why `parent::buildForm` is required in save?

Reply:
This is something to be removed.